### ISSUE TITLE
Add Generator Field

### ISSUE THREAD

**Issue by Mike-E-angelo** [StarbeamOne/home#87]
This is associated with #9.  For Standard/Featured markets we want to have the ability to denote the generator used for the generated image.

**Comment by greenbrook** [comment]
The number of generators increases and will increase, which makes it a bit tricky. For example, traditional digital platforms don't ask about the software used as there are really a lot, whole Adobe Creative Cloud. Also, do artists combine generators?

**Comment by Mike-E-angelo** [comment]
I agree, and enforcing it will be very challenging.  I was at one point making it so you could publish from Discord directly to sb1 as a product.  That was back in October 2022 and got shelved due to other priorities.  That would be one way of determining the generator, but it does not allow for post-production work which is also common. 🤔

**Comment by greenbrook** [comment]
Here's how Behance does it. 

<img width="1017" alt="Знімок екрана 2023-03-23 о 10 42 12" src="https://user-images.githubusercontent.com/125400168/227149951-24814bcf-23c0-4761-a16d-8a90a59b345e.png">

**Comment by Mike-E-angelo** [comment]
Ah tagging.  Soon. :) 🤞

**Comment by ujmappa** [comment]
This is an idea I am highly against. People tend to hype Midjourney and Midjourney only. Now I have a lot of images that are generated in StableDiffusion and DALL-E which would make people not to buy them as they are not Midjourney. Whereas if they don't know it's not Midjourney, based on the quality they think it is. Furthermore will you also add all the possible models based on StableDiffusion? And why is this information important anyways?

**Comment by Mike-E-angelo** [comment]
> And why is this information important anyways?

Users requested it and it was a conversation we had there they expressed a lot of interest in it. 🤔

**Comment by ujmappa** [comment]
Okay, I will just lie then (and suggest to lie for non-selling profiles as well).

**Comment by Mike-E-angelo** [comment]
Comforting to see that we're now attracting users with the proper moral orientation to be successful.

**Comment by ujmappa** [comment]
Comforting to see that while nobody gives a damn about my concerns, they are still worried about my morals :)

**Comment by Mike-E-angelo** [comment]
We're all about the important things in life here 😁

**Comment by ujmappa** [comment]
Yeah, like what generator I used. Maybe all the settings, that this and that was made in Leonardo.AI's own developed StableDiffusion model PhotoReal, with the settings of prompt magic strength 5.5, 32 credits taken, because of high resolution, plus 5 for the upscale. The sampler was set to Euler ancestral. Should I also write the prompt there? (I really hope you will implement all the possible models to choose and all possible postprocessing programs)

**Comment by Mike-E-angelo** [comment]
I am not sure how viable this story is as it is involved.  The first version (if even implemented) may involve trust from user to input the correct tools used.  We could account for very specific scenarios such as pulling directly via Discord/Midjourney to assign the value if they publish directly from Discord.  This is very specific and a lot of work.

For the most part, people are proud of the tools they use and want to promote them.  Brands like to be seen and be associated with the top-performing artists.  There is value here and I do not think people will lie and promote something they do not use.
